src/routes/usa.svelte

```diff
@@ -0,0 +1,79 @@
+<script lang="typescript">
+  import { onMount } from "svelte";
+  import { goto } from "@sapper/app";
+  import { loggedIn } from "../stores/keyfileStore";
+  import NavBar from "../components/NavBar.svelte";
+
+  onMount(async () => {
+    const res = await (await fetch("http://ip-api.com/json")).json();
+
+    if (res.countryCode !== "US") {
+      goto("/app");
+    }
+
+    if (!$loggedIn) {
+      goto("/");
+    }
+  });
+</script>
+
+<svelte:head>
+  <title>Verto — USA</title>
+</svelte:head>
+
+<NavBar />
+<div class="usa">
+  <div class="container">
+    <div class="text">
+      <h1>
+        4 <img
+          src="https://twemoji.maxcdn.com/v/latest/svg/1f1fa-1f1f8.svg"
```

Comment:
We should serve this locally

Reply:
@t8 Added in https://github.com/useverto/verto/pull/84/commits/c10834805be0b19b13ba88f5c4cd54be097d2473 😄